clinical/dbaccess.py

```diff
@@ -13,112 +13,6 @@
 import psutil
 import subprocess
 
-def readconfig( config ):
-  """Reads parameters from a config file.
-
-  Args:
-    config (str): path to config file
-    If config does not exist the default will be used
-  
-  Returns:
-    dict: parameters from the config file (unparsed)
-  """
-  if os.path.isfile(config):
-    Configfile = config
```

Comment:
By convention capitalized variable names are usually reserved for classes in Python. Most variables in Python are lowercase and use underscores for clarity (classes use capitalization + camel case).

```
class Person(object):
  pass
```